egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -37,16 +52,21 @@ if [ $stage -le 1 ]; then
     else
       num_reps=1
     fi
-    reverb_data_dirs=
-    for i in `seq 1 $num_reps`; do
-      cur_dest_dir=" data/temp_${data_dir}_${i}"
-      local/multi_condition/reverberate_data_dir.sh --random-seed $i \
-        --snrs "$snrs" --log-dir exp/make_corrupted_wav \
-        data/${data_dir}  data/impulses_noises $cur_dest_dir
-      reverb_data_dirs+=" $cur_dest_dir"
-    done
-    utils/combine_data.sh --extra-files utt2uniq data/${data_dir}_rvb $reverb_data_dirs
-    rm -rf $reverb_data_dirs
+
+    python steps/data/reverberate_data_dir.py \
+      --prefix "rev" \
+      --rir-set-parameters "1.0, SIM_RIR/simulated_rir_mediumroom/rir_list" \
```

Comment:
Actually I think writing an example script which can be easily copied into other recipes is better. So you could create `local/reverberate_data_dir.sh` script where you download all the necessary databases, combine the different lists with different weights and finally reverberate the data directory.

You can call this script in  `local/nnet3/run_ivector_common.sh`.